packages/ui-components/src/assets/styles/kv-theme.default.scss

```diff
@@ -22,22 +22,24 @@ $warning-color: #ffc043 !default;
 $error-color: #e11900 !default;
 
 // Default Text and Background Colors
-// -------------------------------------------------------------------------------------------
-// Used to calculate the default neutral colors
-$background-color: #fff !default;
-$text-color: #202020 !default;
+$background-color: #202020 !default;
+$text-color: #fff !default;
 
 // Default Neutral Colors
 // -------------------------------------------------------------------------------------------
-// Neutral Color are used to provide variations in text and background colors of elements.
-// Should be $background-color stepping towards $text-color.
+// Neutral Color are used to provide variations in white and black colors of elements.
+// Should be $neutral-color-0 stepping towards $neutral-color-9.
 // `-rgb` prop is used to apply transparencies with RGBA
-$neutral-color-1: mix($text-color, $background-color, 6%);
-$neutral-color-2: mix($text-color, $background-color, 11.5%);
-$neutral-color-3: mix($text-color, $background-color, 15.1%);
-$neutral-color-4: mix($text-color, $background-color, 29%) !default;
-$neutral-color-5: mix($text-color, $background-color, 64%) !default;
-$neutral-color-6: mix($text-color, $background-color, 86%) !default;
+$neutral-color-0: #fff;
+$neutral-color-9: #121212;
+$neutral-color-1: mix($neutral-color-9, $neutral-color-0, 5.5%) !default; // #f2f2f2
```

Comment:
You didn't need calculate the colors with `mix` the color can be fixed this is only a helper if we swap the base colors

Reply:
Yes, I see your point. I meant to leave these calculations here just in case we might need them in the future. I'll manually set all colours.